src/types/types.tsx

```diff
@@ -1,27 +1,27 @@
-import firebase from "firebase/app";
+import firebase from 'firebase/app';
 
 export type User = {
-    id: string;
-    access: string;
-    createdJobs: Job[];
-    email: string;
-    likedJobs: Job[];
-    name: string;
-    phoneNumber: string;
-    verified: boolean;
-    // note: is password necessary here or in firebase?
-    password: string;
-}
+  id: string;
+  access: string;
+  createdJobs: string[]; //switched to string of jobIds to match Firebase
+  email: string | null;
+  likedJobs: string[]; //switched to string of jobIds to match Firebase
+  name: string;
+  phoneNumber: string | null;
+  verified: boolean;
+  // note: is password necessary here or in firebase?
+  password: string | null;
+};
 
 export type Job = {
-    contact_info: string;
-    description: string;
-    employer: string;
-    // note: look into what type the date should be in
-    end_date: Date;
-    hours: number;
-    // note: maybe set type to User
-    job_creator: string;
-    salary: number;
-    start_date: string;
-}
\ No newline at end of file
+  contact_info: string;
+  description: string;
+  employer: string;
+  // note: look into what type the date should be in
```

Comment:
remove comments if unnecessary

Reply:
done (left the note for date type because that could become a concern for post firebase queries)